php_phongo.c

```diff
@@ -907,6 +907,10 @@ int phongo_execute_command(mongoc_client_t *client, php_phongo_command_type_t ty
 			bson_append_bool(&initial_reply, "tailable", -1, 1);
 		}
 
+		if (command->batch_size) {
+			bson_append_int32(&initial_reply, "batchSize", -1, command->batch_size);
```

Comment:
On second thought, let's use `bson_append_int64()` here. The corresponding documentation in libmongoc ([`find_with_opts()`](http://mongoc.org/libmongoc/current/mongoc_collection_find_with_opts.html)) says "batchSize" and "maxAwaitTimeMS" are both "non-negative int64". Logic in `Command.c` already ensures they fall between `0` and `UINT32_MAX` (inclusive), so we don't have to worry about that. Assigning as an int64 will also ensure that `UINT32_MAX` will not be interpreted as a negative int64 down the line (an edge case, I know).

I'd also suggest changing the append function for "maxAwaitTimeMS" a few lines up in a second commit.

Reply:
On that note, should we make the new ``batch_size`` member of the struct an ``uint64_t`` instead of an ``uint32_t`` below too then?

Reply:
`uint32_t` should cast up to `uint64_t` without issue and we only store these values if they call within the unsigned 32-bit range. I think this is OK to remain as-is.